zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/schema/SchemaManager.java

```diff
@@ -89,13 +109,21 @@ private void initialiseIndexTemplates() {
             .getMappings(config.getIndex().getPrefix() + "*", MappingSource.INDEX_TEMPLATE)
             .keySet();
 
+    LOG.info(
+        "Create missing index templates. [existing: {}, missing: {}]",
+        existingTemplateNames.size(),
+        indexTemplateDescriptors.size() - existingTemplateNames.size());
```

Comment:
This existing template names is always going to be the wrong size for an empty index prefix. It includes templates created by ES/OS such as `.monitoring-es`. If the prefix is always going to be set to a value this is irrelevant but if it defaults to an empty prefix then we can just add a `filter(name -> !name.startsWith(".")` should be good.

Reply:
We could also change this in the `getMappings` to where we automatically filter out ES/OS created templates

Reply:
Could we also use a regex like `^[a-zA-Z]+.*` ? 🤔

Reply:
that could work as well